src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -271,15 +272,39 @@ public FeedbackSessionsCopyFromModal getCopyFromModal() {
             }
         }
         
-        // Add "No active courses" option if there are no active courses
-        if (result.isEmpty()) {
-            ElementTag blankOption = createOption("No active courses!", "", true);
-            result.add(blankOption);
+        // if courses is not empty but result is empty, means the instructor have no permission to edit
+        // rather than the instructor have no active courses
+        // add "No active courses" option if there are no active courses
+        ElementTag defaultOption;
+        if (result.isEmpty() && !courses.isEmpty()) {
+            defaultOption = createOption("No permission to modify courses!", "", true);
+            result.add(defaultOption);
+        } if (result.isEmpty()) {
```

Comment:
The behaviour's probably the same, but is there supposed to be a newline or an `else` here?
That said, this would be best represented with a nested `if` for the courses empty check.

Also, if you do intend to have this other option, do provide a more accurate message.
It's the lack of permission to modify the courses' sessions, rather than the courses themselves.

I do wonder if it's better to use a single message for both cases, though, like maybe "No usable courses found!".
The instructor should have the same capabilities in either case.
This can be a new issue.